### PR TITLE
add Complex Radio

### PR DESCRIPTION
**Pull request by TylerCousins**

### CODE REVIEW

File: Complex Radio/presence.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    const data = await window
+        .fetch("https://radio.complexr.pw/api/nowplaying/1")
+        .then((res) => res.json());
+    ssong = data.now_playing.song.text;
+    spresenter = data.live.is_live ? "Presenter " + data.live.streamer_name : "AutoDJ";
+    slisteners = data.listeners.unique + " Listeners";
+}
+
+setInterval(newStats, 2500);
+newStats();
```
Comment (Alanexei) [inline Complex Radio/presence.ts:15]: You are calling an async function without await. That's the problem.

Comment (TylerCousins) [reply Complex Radio/presence.ts:15]: Right how would i fix this. As it seems to be fine when i load the presence

Comment (Alanexei) [reply Complex Radio/presence.ts:15]: It's part of the reason that you are getting the DeepScan issue.

Comment (TylerCousins) [reply Complex Radio/presence.ts:15]: https://i.imgur.com/LevJzfA.png

Comment (TylerCousins) [reply Complex Radio/presence.ts:15]: still dk how to fix that deepscan

Comment (Alanexei) [inline Complex Radio/presence.ts:15]: This is supposed to be `await newStats()`

Comment (Alanexei) [reply Complex Radio/presence.ts:15]: Send the screenshot directly to github by copy and pasting it into a comment.

Comment (TylerCousins) [reply Complex Radio/presence.ts]: done that still issue with the deepsync

**Pull request closed without merging**